src/utils/prices.ts

```diff
@@ -2,8 +2,19 @@ import { Token } from 'uniswap-xdai-sdk'
 
 import { BigNumber } from '@ethersproject/bignumber'
 
+import { STABLE_TOKEN_ADDRESSES } from '../constants/config'
 import { tryParseAmount } from '../state/orderPlacement/hooks'
 
+export function shouldPricesBeInverted(auctioningTokenAddress: string): boolean {
+  return STABLE_TOKEN_ADDRESSES.includes(auctioningTokenAddress)
+}
+
+export function getInverse(price: number, digits: number): number {
+  const re = new RegExp('(\\d+\\.\\d{' + digits + '})(\\d)'),
+    m = (1 / price).toString().match(re)
+  return m ? parseFloat(m[1]) : (1 / price).valueOf()
+}
+
```

Comment:
This takes the first matching group of a decimal number with a certain amount of digits after a period and based on that does (1/price) two times (giving back price again) or not?.  Maybe by looking at it more time I can have a better understanding of what it is doing.

Some ideas
- Leave a comment about the two capture groups of the regexp or give them a name, `const [foo, bar] = m` or ?<name> but the latter make it harder to read IMHO.
- Leave a comment about what it means to not have a match, maybe a numeric example (or more than one).
- Some unit testing, will this always work for javascript float and number precision?

Reply:
Yes, unit tests are the best. I will do it later, as I still want to do the release today